### PR TITLE
[TASK] Add General Recommendation section for Permissions Managemend docs

### PR DESCRIPTION
**Pull request by soee**
This commit adds the General Recommendations section to the new
Permissions Management documentation.

This is part 2 from the PR series.

### CODE REVIEW

**Comment by linawolf** [comment]
@soee please resolve conflicts

**Comment by soee** [comment]
> @soee please resolve conflicts

@linawolf i have rebased on `main`, conflicts should be resolved.

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
+.. include:: /Includes.rst.txt
+
+.. index:: backend, acl, permissions, user groups, user management
+
+.. _general-recommendations:
+
+=======================
+General Recommendations
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:8]:
```suggestion
General recommendations
```

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+.. _user-specific-accounts:
+
+Create user-specific accounts
+=============================
+
+These users should not have generic usernames set like editor, webmaster, proofreader, etc.
+Instead, you should use their real names, such as :samp:`johndoe`, :samp:`john.doe`, or even
+their email address, :samp:`john.doe@mail.com`. This approach ensures you always know
+which user logged into the TYPO3 backend and who is responsible for specific
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:25]: Try to use shorter sentences they are easier to follow

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+.. note::
+    Avoid generic names for a backend username, instead use their real names.
+
+.. _ensure-safety:
+
+How to ensure safety
+====================
+
+When setting up permissions in TYPO3, it's really important to start off
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:48]: Please also change this in other places

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+by giving users just the access they actually need, and only add more
+if it turns out they need it. This helps keep things safe. It's also a good idea
+not to hand out admin rights unless there's no other option. For everyday tasks
+like managing content, it's better to use special accounts set up just for those jobs.
+
+If someone's only supposed to have access to the TYPO3 backend for a little while,
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:54]: In general all of this is much more text then readers will probably read. Try to make it shorter, short sentences, bullet points, something quick to grasp?

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    - Instead of using administrative accounts, create groups for specific roles, such as editor, and assign these roles to users
+    - Regularly maintain backend user accounts, removing any that are no longer needed
+    - Always establish secure passwords for users
+
+.. _permissions-via-groups:
+
+Set permissions via groups, not user records
+============================================
+
+Some permissions or access rights can be directly configured through the backend
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:80]: Again I think this is way to much text and not easy to read

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+.. note::
+    Configure permissions only through the backend user groups. Don’t set them on the user record.
+
+.. _file-mounts-and-files-management:
+
+File mounts and files management
+================================
+
+In the previous section, we described naming conventions for backend groups,
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:112]: Do not refer to previous section, you are not writing a book, ppl jump on the web ;-)

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+within a role group, ensuring that each user associated with such a role group
+will have access to the specified folders and files in the storage.
+Let's look at an example.
+
+..  uml:: _sample_backend_groups_hierarchy.plantuml
+    :align: center
+    :caption: Sample backend groups hierarchy
+    :width: 700
+
+This is a simple diagram illustrating how you can structure your folders in storage,
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:134]: Never assume anything is simple, this leads to frustrations with newbies

File: Documentation/PermissionsManagement/GeneralRecommendations/Index.rst [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+create dedicated File Mounts for them, and then use those File Mounts within backend user groups.
```
Comment (linawolf) [inline Documentation/PermissionsManagement/GeneralRecommendations/Index.rst:135]:
```suggestion
create dedicated file mounts for them, and then use those file mounts within backend user groups.
```

**Review by linawolf — COMMENTED** [review]
Thanks for your work in this, I have some small remarks, expecially try to make shorter sentences, remove rethorical "fluff", use more headlines, sections, bullet points and other structuring elements. Introduce a table of contents (directive .. contents::) so users can jump quickly to the information they need

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
Great work, thanks

**Comment by linawolf** [comment]
@soee please resolve conflicts

**Review by linawolf — APPROVED** [review]

**Pull request merged**